Formula/mruby.rb

```diff
@@ -19,6 +20,7 @@ class Mruby < Formula
   uses_from_macos "ruby"
 
   def install
+    inreplace "build_config.rb", /default/, "full-core"
```

Comment:
Why is this change needed?

This looks unrelated to `qt`, so please drop it from this PR and open a separate one for it.

Reply:
qsoas depend on qt and mruby, qsoas need full-core version of mruby